mdakits/mdance/metadata.yaml

```
@@ -0,0 +1,108 @@
+## Please note these _must_ be GitHub handles
+## The maintainers will be tagged in issues if their MDAKit is failing.
+maintainers:
+  - lexin-chen
+
+## str: a free form description of the mdakit
+description:
+    flexible n-ary clustering package for Molecular Dynamics trajectories.
+
+## List(str): a list of keywords which describe the mdakit
+keywords:
+  - clustering
+  - machine-learning
+  - algorithms
+  - molecular-dynamics
+  - protein-refinement
+
+## str: the license the mdakit falls under
+## See https://spdx.org/licenses/ for valid license specifiers
+license: MIT
+
+## str: the link to the project's code
+## Please note that this is not limited to GitHub! Can be Gitlab, etc..
+project_home: https://github.com/mqcomplab/MDANCE
+
+## str: the link to the project's documentation
+documentation_home: https://mdance.readthedocs.io
+
+## str: the type of documentation available [UserGuide, API, README]
+documentation_type: UserGuide + API
+
+#------------------------------------------------------------
+# Optional entries
+#------------------------------------------------------------   
+## List(str): a list of commands to use when installing the latest
+## release of the code. Note: only one installation method can currently
+## be defined. We suggest using mamba where possible (e.g.
+##   mamba -c conda-forge install MYPROJECT
+## for a conda package installation).
+## Here we use a simple PyPi installation:
+install:
+  - pip install mdance
+
+## List(str): a list of commands to use when installing the mdakit from its
+## source code.
+src_install:
+  - pip install git+https://github.com/mqcomplab/MDANCE@main
+
+## str: the package name used to import the mdakit
+import_name: mdance
+
+## str: a specification for the range of Python versions supported by this MDAKit
+python_requires: ">=3.9"
+
+## str: a specification for the range of MDAnalysis versions supported by this MDAKit
+mdanalysis_requires: ">=2.0.0"
+
+## List(str): a list of commands to use when attempting to run the MDAKit's tests
+## If you package your tests inside your package then you can typically use the 
+##     pytest --pyargs MYPACKAGE
+## command as shown below. 
+## Otherwise you need to include commands to make the tests available. 
+## For example, if the tests are in the repository at the top level under `./tests`:
+## First use `git clone latest` to either clone the top commit for "development code" checks or check out
+## the latest tag for "latest release" checks. Then then run pytest:
+##    - git clone latest
+##    - pytest -v ./tests
+## Feel free to ask for advice on your pull request!
+#run_tests:
+  - pytest ./tests
+
+## List(str): a list of commands to use to install the necessary dependencies required
+## to run the MDAKit's tests.
+## The default below _might_ be sufficient or you might not even need MDAnalysisTests:
+## make sure that it is appropriate for how you run tests.
+#test_dependencies:
+#  - mamba install pytest MDAnalysisTests

```

We also need to install pytest because it's not listed in the dependencies https://github.com/mqcomplab/MDANCE/blob/9791f82a872b2f677f44391cf2998b4a2c143005/pyproject.toml#L19-L25 that are installed with the `src_install:` commands.
```suggestion
test_dependencies:
  - pip install pytest
```